backend/config/initializers/rswag-ui.rb

```diff
@@ -0,0 +1,10 @@
+Rswag::Ui.configure do |c|
```

Comment:
You went all the way to import this gem but did not document any of the controllers?